.travis.yml

```diff
@@ -50,19 +55,13 @@ before_script:
   - npm --prefix api install
   - npm --prefix sentinel install
   - ./node_modules/.bin/webdriver-manager update
-  - ./node_modules/.bin/webdriver-manager start &
+  - ./node_modules/.bin/webdriver-manager start >/dev/null &
```

Comment:
We don't want to commit this, either leave as is or use this as an excuse to push it to a file and upload that file to S3

Reply:
Yes, sorry!  Should be another TODO for this at the top of the PR - I deleted this stuff to make failures clearer while I was developing.